docs/config/lua/wezterm/on.md

```diff
@@ -22,6 +22,8 @@ There is no way to de-register an event handler.  However, since the Lua
 state is built from scratch when the configuration is reloaded, simply
 reloading the configuration will clear any existing event handlers.
 
+See [wezterm.action_callback](./action_callback.md) for a helper to define a custom action callback.
```

Comment:
What do you think about adjusitng the custom events example below to use the new helper?

Reply:
I'm not sure, I think it's important to have an example using `wezterm.on` and `wezterm.action{EmitEvent=...}` explicitely, in case one wants to do something specific that would not be supported by `wezterm.action_callback` (like using the event id in another custom action, to emit it manually with `wezterm.emit(...)`).

I can write another simpler example below it though (the same as in the docs for `wezterm.action_callback` ?) to show its usage in this doc file?

Reply:
How about both in the same example?  If the line starting `wezterm.on("trigger-vim-with-scrollback", function(window, pane)` becomes a helper function definition, rather than a closure: `function trigger_vim_with_scrollback(window, pane)`,
then we could show the new `action_callback` example first:

```lua
return {
  keys = {
    {key="E", mods="CTRL",
      action=wezterm.action_callback(function(win, pane)
           trigger_vim_with_scrollback(win, pane)
      end)
    }
}
```

then show the older/discrete way as an alternative:

```lua
wezterm.on("trigger-vim-with-scrollback", trigger_vim_with_scrollback)
return {
  keys = {
    {key="E", mods="CTRL",
      action=wezterm.action{EmitEvent="trigger-vim-with-scrollback"}},
  }
}
```